src/lib/serviceRoutinesV2/getEntityAttributeValue.cpp

```diff
@@ -56,6 +55,7 @@ std::string getEntityAttributeValue
 {
   std::string  answer;
   Attribute    attribute;
+  bool         text = (ciP->uriParam["options"]=="text" || ciP->httpHeaders.accept == "text/plain");
```

Comment:
style: spaces for '==':

    ciP->uriParam["options"] == "text"

Then a doubt ...
ciP->httpHeaders.accept could easily be "application/xml, test/plain", or any other combination. The charset could be there also: text/plain;charset=UTF8 (I don't remember exactly the syntaxis here ...).
This comparison is not enough for all cases.
But, this should be taken care of somewhere ... somewhere in rest.cpp.
Can't you use the result of that 'thing' instead of this simple comparison?